packager/media/codecs/h264_byte_to_unit_stream_converter_unittest.cc

```diff
@@ -108,20 +108,43 @@ TEST(H264ByteToUnitStreamConverter, NaluConversionWithSpsExtension) {
       0x00, 0x01, 0x68, 0xFE, 
       0x00, 0x00, 0x00, 0x01, 	// Start code
       // SPS Extension data
-      0x00, 0x05, 0x6d, 0x33, 0x01, 0x57, 0x78
+      0x00, 0x05, 0x6D, 0x33, 0x01, 0x57, 0x78
   };
 
-  int kSpsExtSize = arraysize(kSpsExtArr);
-  std::vector<uint8_t> kSpsExt(kSpsExtArr, kSpsExtArr + kSpsExtSize);
+  std::vector<uint8_t> byte_stream_with_sps_extension(std::begin(kByteStreamWithSpsExtension), 
+  						      std::end(kByteStreamWithSpsExtension));
   H264ByteToUnitStreamConverter converter(
-      H26xStreamFormat::kNalUnitStreamWithParameterSetNalus);
+      H26xStreamFormat::kNalUnitStreamWithoutParameterSetNalus);
 
-  std::vector<uint8_t> unit_stream_output;
+  std::vector<uint8_t> unit_stream;
   ASSERT_TRUE(converter.ConvertByteStreamToNalUnitStream(
-              kSpsExt.data(), kSpsExt.size(),
-              &unit_stream_output));                                                 
-  std::vector<uint8_t> byte_stream_output;
-  EXPECT_TRUE(converter.GetDecoderConfigurationRecord(&byte_stream_output));
+              byte_stream_with_sps_extension.data(), byte_stream_with_sps_extension.size(),
+              &unit_stream));   
+              
+  const uint8_t kExpectedDecoderConfig[] = {
+      0x00, 0x00, 0x00, 0x0A, 0x06, 0xFD, 0x78, 0xA4, 0xC3, 
+      0x82, 0x62, 0x11, 0x29, 0x77, 0x00, 0x00, 0x00, 0x04, 
+      0x00, 0x02, 0x67, 0x64, 0x00, 0x00, 0x00, 0x04, 0x00, 
+      0x01, 0x68, 0xFE, 0x00, 0x00, 0x00, 0x07, 0x00, 0x05, 
+      0x6D, 0x33, 0x01, 0x57, 0x78};          
+  EXPECT_EQ(std::vector<uint8_t>(kExpectedDecoderConfig,
+                                 kExpectedDecoderConfig + arraysize(kExpectedDecoderConfig)),
```

Comment:
```c++
  EXPECT_EQ(std::vector<uint8_t>(std::begin(kExpectedDecoderConfig),
                                 std::end(kExpectedDecoderConfig)),
```

Same below.

Reply:
done, i think this or something similar was fixed on the 2nd most recent commit